guides/screen-reader-differences.md

```diff
@@ -0,0 +1,134 @@
+In this guide:
+
+- [Screen reader overview](#Overview)
+  - [Commonalities](#Commonalities)
+  - [Differences](#Differences)
+      - [Interaction modes](#Interaction-modes)
+- [Guidelines for developers](#Guidelines-for-developers)
+
+# Overview
+
+Screen reader software allows users to navigate webpages by having the text of the page read aloud to them linearly. Screen reader users can also interact with the elements on the page using special keyboard commands that are unique to each application. The top 3 screen readers according to the [2017 WebAIM Survey](https://webaim.org/projects/screenreadersurvey7/) are JAWS, NVDA, and VoiceOver:
+
+|  Screen Reader |  % of respondents who commonly use it | 
+|---|---|
+|  JAWS | 66.0%  |
+|  NVDA | 64.9%  |
+|  VoiceOver | 39.6%  |
+
+JAWS and NVDA are Windows applications. VoiceOver is available as a desktop Mac OS application and on iOS devices. If you are new to screen readers or need a refresher, check out this [demo video of how a screen reader works](https://www.youtube.com/watch?v=q_ATY9gimOM).
+
+
+## Commonalities
+
+It is important to keep in mind that screen readers "present content linearly to users, one item at a time. This contrasts with the way in which most people use visual interfaces....Users must progress through such systems in a step-wise manner. The insight that audio interfaces are linearized versions of web content is an important one that should guide web developers during the engineering and design process." [Source](https://webaim.org/techniques/screenreader/)
```

Comment:
Four-dot ellipsis should be formatted with spaces around the ellipse (after the period and before the next sentence).

```This contrasts with the way in which most people use visual interfaces. ... Users must progress through such systems in a step-wise manner.```